### DISCUSSION TITLE
Filter habit list based on priority (or urgency), or at least the option to show an indicator

### DISCUSSION THREAD

**Discussion by drawtheliner** [iSoron/uhabits#1378] [Feature Requests]
I understand there is another request related to this, but in regards to sorting (I just did [a comment there](https://github.com/iSoron/uhabits/discussions/921#discussioncomment-2769473)).

However I'd rather keep my habits list always in the same order (and, if possible, [grouping](https://github.com/iSoron/uhabits/discussions/495)). That way I could just **filter** the list by priority.

Or/also, to have an icon/indicator next to each habit showing its priority (maybe as icons - red/yellow/green).

Not sure if the sorting by priority (currently called "status") is working though, as I mention in the first comment linked above.

**Comment by hiqua** [comment]
As I wrote in my other comment, please expand on what you mean by priority. There are already colors as well.

**Reply by drawtheliner** [reply]
Thanks! I tried to explain it better [there](https://github.com/iSoron/uhabits/discussions/921#discussioncomment-2777357).
